docs/endpoints/multi_user.rst

```diff
@@ -1042,6 +981,231 @@ make the necessary changes to ``config.yaml``:
+       --auth-policy-project-id ba1e3f22-02d4-477b-b5d0-f8af2e8c419c \
+       --allowed-domains "globus.org" \
+       --auth-timeout 1800 \
+      ha_documentation_compute_endpoint
+
+This example command line will *create* a new HA (``--high-assurance``) policy under
+the ``ba13...`` project that will require users be from the ``globus.org`` domain and
+that they authenticate every ``1800`` seconds.
+
+This example does not show all options; use ``configure --help`` to see what is
+available:
+
+.. code-block:: console
+
+   # globus-compute-endpoint configure --help
+   Usage: globus-compute-endpoint configure [OPT...
+
+
+High-Assurance Functions
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+A High-Assurance function is one that has been registered with the Globus Compute API
+as associated with a HA endpoint.  In other words, a HA function requires (exactly one)
+HA endpoint.  A HA function may not be run on any other endpoint and will be removed
+from all Globus-operated storage after 3 months of inactivity.
+
+To register a High-Assurance function, specify the ``ha_endpoint_id`` argument to
+``register_function()``:
+
+.. code-block:: python
+
+   from globus_compute_sdk import Client, Executor
+
+   def ha_func(pii: str):
+      from datetime import datetime
+      return f"ha_func: {datetime.now()} - processed pii: {pii}"
+
+   gc = Client()
+   ex = Executor()  # for illustrative purposes; strongly consider `with Executor() as ex:` style
+
+   # Registration works the same, whether via the Client or the Executor
+   ha_func_id_via_client = gc.register_function(ha_func, ha_endpoint_id='...')
+   ha_func_id_via_ex = ex.register_function(ha_func, ha_endpoint_id='...')
+
+   # Now save one of the registered ids (from this example, they point to the same
+   # function logic) for later use with `.submit_to_registered_function()`
+   print(ha_func_id_via_ex)
+
+Further, SDK usage of a HA function will require the SDK interaction to follow the HA
+requirements of the associated policy.  For example, a long running HA task (a task
+that uses an HA function) might require the SDK to login again before downloading the
+associated result:
+
+.. code-block:: python
+   :emphasize-lines: 6,15
+   :linenos:
+
+   >>> from globus_compute_sdk import Executor
+   >>> ha_endpoint_id = '...'
+   >>> ha_function_id = '...'
+   >>> with Executor(endpoint_id=ha_endpoint_id) as ex:
+   ...     fut = ex.submit_to_registered_function(ha_function_id, "ha-worthy-argument-to-function")
+   ...     print("\nResult:", fut.result())
+
+   Please authenticate with Globus here:
+   -------------------------------------
+   https://auth.globus.org/v2/oauth2/authorize?...
+   -------------------------------------
+
+   Enter the resulting Authorization Code here: ...
+
+   It is your responsibility to ensure disclosure of regulated data, such as Protected Health Information, resulting from the submission of this function and its arguments is legally authorized.
```

Comment:
Consider making this an `attention` admonition.

Reply:
This is "the raw output."  I can call it out again in a separate admonition, but it's not text I originated.  The point of including the naked output (sans redaction ellipses) was to give an honest console interaction for general context of what to expect.